<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestao_vendas</class>
 <widget class="QDialog" name="gestao_vendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateEdit" name="de_ini">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="date">
    <date>
     <year>2021</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="de_f">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Inicial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Final</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_filtrar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_vendas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1180</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_prod_v">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>1181</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pdf">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar PDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_all">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostrar Todas as Vendas</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_clientes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>1181</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_excluir">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>580</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Excluir Venda</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar Venda</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
